datasetinsights/constants.py

```diff
@@ -36,7 +36,7 @@
 #   ...
 DEFAULT_DATA_ROOT = "/data"
 SYNTHETIC_SUBFOLDER = "synthetic"
-DEFAULT_PUBLIC_DATASET = "synthetic"
+DEFAULT_PUBLICDEFAULT_KFP_LOG_DIR_DATASET = "synthetic"
```

Comment:
It's not clear where this constant is used. Also, the original constant variable `DEFAULT_PUBLIC_DATASET` is very confusing. Can it this be safely removed? Why is this change required?